### PR TITLE
DAOS-6900 test: Fix avocado monkey patching

### PR DESCRIPTION
**Pull request by brianjmurrell**
To work with 69.0 also.

### CODE REVIEW

File: src/tests/ftest/harness/basic.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
```
Comment (phender) [inline src/tests/ftest/harness/basic.py:1]: Don't we need the license?
```suggestion
#!/usr/bin/python
"""
  (C) Copyright 2018-2021 Intel Corporation.
  SPDX-License-Identifier: BSD-2-Clause-Patent
"""
```

**Review by phender — APPROVED** [review]

File: src/tests/ftest/harness/basic.py (hunk 1 above)
Comment (brianjmurrell) [reply src/tests/ftest/harness/basic.py:1]: Yeah.  Added.  Thanks.

**Review by phender — APPROVED** [review]

File: src/tests/ftest/scripts/main.sh [hunk 2]
```diff
@@ -124,13 +125,27 @@ fi
...
-      sed -e '/selftests\/.*/d' |                       \
+    if ! filterdiff -p1 -x selftests/* <                       \
+        "$PATCH_DIR"/avocado-report-test-phases-common.patch | \
+      sed -e '/selftests\/.*/d' |                              \
       sudo patch -p1 -d "$pydir"; then
-        echo "Failed to apply avocado PR-3154 patch"
+        echo "Failed to apply avocado PR-3154 patch - common portion"
         exit 1
     fi
+    if grep "^TEST_STATE_ATTRIBUTES = "
```
Comment (phender) [inline src/tests/ftest/scripts/main.sh:135]: Missing a `\`?
```suggestion
    if grep "^TEST_STATE_ATTRIBUTES = " \
```

Comment (brianjmurrell) [reply src/tests/ftest/scripts/main.sh]: Oops.  Yeah.  Thanks for the heads-up.

**Review by phender — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

**Pull request merged**